rustsec/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.27.0 (TBD)
```

Comment:
Can probably just call this...

```suggestion
## 0.26.1 (TBD)
```

It's a behavior change and not one that concerns end users.

Reply:
Yeah suspected, took the opportunity to fix the link as well :)